Sources/MapboxNavigation/PreviewViewController.swift

```diff
@@ -225,10 +278,22 @@ public class PreviewViewController: UIViewController, BannerPresentation {
                   popAllExceptFirstBanner: true)
     }
     
-    // :nodoc:
+    /**
+     Adds the specified banner to the banners hierarchy and displays it.
+     
+     - parameter banner: `Banner` instance that will be presented.
+     - parameter animated: Controls whether `PreviewViewController` animates the transition
+     between banners. Defaults to `true`.
+     - parameter duration: Duration of the animation. Defaults to `0.2` seconds.
+     - parameter animations: A block object containing the changes to commit to the views.
+     This is where you programmatically change any animatable properties of the views in your
+     view hierarchy. This block takes no parameters and has no return value.
+     - parameter completion: A block object to be executed when the animation sequence ends.
+     This block has no return value.
+     */
     public func present(_ banner: Banner,
                         animated: Bool = true,
-                        duration: TimeInterval = 1.0,
+                        duration: TimeInterval = 0.2,
```

Comment:
If we think this will be a common animation duration, let’s factor it out into a global in the constants file.

Reply:
Still unsure whether it'll be common across the entities, so will probably hold off about its extraction into separate property, as it'll have to be public.